apps/callscreen/index.html

```diff
@@ -17,20 +17,18 @@
 
     <!-- Localization -->
     <script defer type="text/javascript" src="/shared/js/lazy_loader.js"></script>
-    <script defer type="application/javascript" src="/shared/js/lazy_l10n.js"></script>
     <meta name="defaultLanguage" content="en-US">
     <meta name="availableLanguages" content="en-US">
+    <link rel="localization" href="callscreen/locales/callscreen.{locale}.properties">
     <link rel="localization" href="../communications/dialer/locales/dialer.{locale}.properties">
     <link rel="localization" href="../communications/dialer/locales/shared.{locale}.properties">
     <link rel="localization" href="shared/locales/date/date.{locale}.properties">
-    <link rel="localization" href="shared/locales/phone_types/phone_types.{locale}.properties">
     <link rel="localization" href="shared/locales/lockscreen/lockscreen.{locale}.properties">
     <link rel="localization" href="shared/locales/keypad/keypad.{locale}.properties">
     <link rel="localization" href="shared/locales/telephony/telephony.{locale}.properties">
-    <!-- Making sure the shared resources will be copy into the zip
-    <script type="application/javascript" src="/shared/js/l10n.js"></script>
-    <script type="application/javascript" src="/shared/js/l10n_date.js"></script>
-    -->
+
+    <script defer type="application/javascript" src="/shared/js/l10n.js"></script>
+    <script defer type="application/javascript" src="/shared/js/l10n_date.js"></script>
```

Comment:
can you remove this file in your patch? I removed all uses and forgot to remove the reference :)